### ISSUE TITLE
Error in setting up dependencies

### ISSUE THREAD

**Issue by jaredbernard** [oushujun/EDTA#209]
Hi, @oushujun.

I have installed EDTA according to your instructions, but when I run the 5-min test genome as instructed, a set of error messages indicates there is a problem with `biopython` and `pandas`. I have tried uninstalling and reinstalling each several times (both via `base` and `EDTA` envs), and running the associated import commands (`python >>> from Bio import SeqIO \ import pandas as pd`), but the problem remains.

Here are the specific error messages:
```
Species: others
Traceback (most recent call last):
  File "/usr/local/EDTA/bin/TIR-Learner2.5/pre.py", line 1, in <module>
    from Bio import SeqIO
ModuleNotFoundError: No module named 'Bio'
Traceback (most recent call last):
  File "/usr/local/EDTA/bin/TIR-Learner2.5/Module2/RunGRF.py", line 1, in <module>
    from Bio import SeqIO
ModuleNotFoundError: No module named 'Bio'
cp: cannot stat 'TIR-Learner/TIR-Learner*': No such file or directory
Traceback (most recent call last):
  File "/usr/local/EDTA/bin/TIR-Learner2.5/Module2/ProcessGRFmite.py", line 1, in <module>
    from Bio import SeqIO
ModuleNotFoundError: No module named 'Bio'
cp: cannot stat 'TIR-Learner/*-p': No such file or directory
Traceback (most recent call last):
  File "/usr/local/EDTA/bin/TIR-Learner2.5/Module3_New/getDataset.py", line 10, in <module>
    from Bio import SeqIO
ModuleNotFoundError: No module named 'Bio'
Traceback (most recent call last):
  File "/usr/local/EDTA/bin/TIR-Learner2.5/Module3_New/CheckTIRTSD_M3.py", line 1, in <module>
    from Bio.Seq import Seq
ModuleNotFoundError: No module named 'Bio'
Traceback (most recent call last):
  File "/usr/local/EDTA/bin/TIR-Learner2.5/Module3/WriteToGff_M3.py", line 1, in <module>
    from Bio import SeqIO
ModuleNotFoundError: No module named 'Bio'
Traceback (most recent call last):
  File "/usr/local/EDTA/bin/TIR-Learner2.5/Module3_New/CombineAll.py", line 1, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
mv: cannot stat '*.gff3': No such file or directory
Traceback (most recent call last):
  File "/usr/local/EDTA/bin/TIR-Learner2.5/Module3/GetAllSeq.py", line 6, in <module>
    from Bio import SeqIO
ModuleNotFoundError: No module named 'Bio'
mv: cannot stat 'TIR-Learner/*FinalAnn*.gff3': No such file or directory
mv: cannot stat 'TIR-Learner/*FinalAnn*.fa': No such file or directory
Can't open ./TIR-Learner-Result/TIR-Learner_FinalAnn.fa: No such file or directory at /usr/local/EDTA/util/rename_tirlearner.pl line 19.
Warning: LOC list genome.fa.mod.TIR.ext30.list is empty.

Error: Error while loading sequenceCan't open ./TIR-Learner-Result/TIR-Learner_FinalAnn.gff3: No such file or directory.
Warning: The TIR result file has 0 bp!

Tue Aug  3 16:56:40 HST 2021    Start to find Helitron candidates.

Tue Aug  3 16:56:40 HST 2021    Identify Helitron candidates from scratch.

Tue Aug  3 16:57:41 HST 2021    Finish finding Helitron candidates.

Tue Aug  3 16:57:41 HST 2021    Execution of EDTA_raw.pl is finished!

ERROR: Raw TIR results not found in genome.fa.mod.EDTA.raw/genome.fa.mod.TIR.raw.fa
        If you believe the program is working properly, this may be caused by the lack of intact TIRs in your genome. Consider to use the --force 1 parameter to overwrite this check
```

Any thoughts? I've tried many things from searches around the web.

**Comment by oushujun** [comment]
There are many similar questions in this repo. You may see how others overcome this. 

My first suggestion: Have you tried installing using the yml file? If failed, then you can try the following things:
1. update conda.
2. unload any modules (`module clear`) before installation
3. use the singularity version

Best,
Shujun

**Comment by jaredbernard** [comment]
Thanks for replying. I finally got EDTA working, although I'm not entirely sure how. Just so you're aware, here is what I tried.

1. I had initially installed via conda and the .yml file, but that didn't help.
2. As noted in my first post, I tried reinstalling biopython and pandas with no luck.
3. I then tried the alternate method used by @wyim-pgl at the bottom of [this thread](https://github.com/oushujun/EDTA/issues/147), but still had the same errors.
4. I also tried following your slightly different installation instructions in [the new tutorial](http://dx.doi.org/10.1007/978-1-0716-1134-0_4), but that didn't appear to work.
5. Then I tried to install via Docker, but despite many attempts and searches online, I couldn't get past the error of the Docker daemon not running.
6. I tried installing Singularity, but also ran into errors.

After all that the original conda installation simply started working! All I can imagine is that rebooting Ubuntu during the Docker attempts triggered the conda version to import Bio and pandas on its own. So far it seems to be working great! I'm currently plugging the output fasta and gff3 into Maker for genome annotation.
